public/index.js

```diff
@@ -0,0 +1,35 @@
+import { gos2desc } from "../src/gos2desc/index.js";
+import { desc2text } from "../src/desc2text/index.js";
+import { desc2treeText } from "../src/desc2tree/treeContent.js";
+import { createFullTree, addToggle } from "../src/desc2tree/createTree.js";
```

Comment:
You could import these three functions directly from the `../src/desc2tree/index.js`. For example, you can add the following lines at the top of `../src/desc2tree/index.js`:

```js
export { desc2treeText } from "./treeContent.js";
export { createFullTree, addToggle } from "./createTree.js";
```

If we publish individual folders as separate libraries later (`desc2tree`, `desc2text`, `gos2desc`), users will be able to import those key functions that are contained in the `index.js` file, e.g.,

```
import { desc2treeText, createFullTree } from 'desc2tree'; // npm package
```

(See Gosling code for instance: https://github.com/gosling-lang/gosling.js/blob/master/src/index.ts.)